### ISSUE TITLE
Amplify build fails. Process is killed with no explanation (Non-Zero Exit Code detected) in the logs.

### ISSUE THREAD

**Issue by hsbNicolas** [aws-amplify/amplify-console#789]
**App Id**:

arn:aws:amplify:eu-west-1:469809149803:apps/d13wpvainbnduk

**Region**: eu-west-1
**Step**:
**Status**:

Terminating logging...
>**Note**: Do not include information that is sensitive in nature such as your domain name, company etc.

2020-06-24T14:43:02.983Z [ERROR]: !!! Build failed
2020-06-24T14:43:03.114Z [ERROR]: !!! Non-Zero Exit Code detected
2020-06-24T14:43:03.115Z [INFO]: # Starting environment caching...
2020-06-24T14:43:03.135Z [INFO]: # Environment caching completed

**Issue/question**
A clear and concise description of what the issue/question is.

The build fails with no explanation of why. The procees is just killed. It always worked fine for many  months. How can we get this fixed?

**Error message**
If there is an error message, please include it here. 

2020-06-24T14:43:02.983Z [ERROR]: !!! Build failed
2020-06-24T14:43:03.114Z [ERROR]: !!! Non-Zero Exit Code detected
2020-06-24T14:43:03.115Z [INFO]: # Starting environment caching...
2020-06-24T14:43:03.135Z [INFO]: # Environment caching completed

**Additional information**
Build is for the branch/release/beta. It builds perfectly fine locally; for all developers. It has been failing since the 12th of June. We use the beta site to test before deploying to production, so its important  for us to get this working.

**Comment by abhi7cr** [comment]
Hi @hsbNicolas ,

From the logs, it looks like the last successful build had @angular/cli@9.1.7 installed . In the failing buildings, I can see either @angular/cli@9.1.8 or 9.1.9 installed. 


Can you check if setting the version explicitly to 9.1.7 solves the issue?

**Comment by hsbNicolas** [comment]
Hello and thanks for the quick reply. I have set the angular/cli to version  9.1.7 explicitly by running npm install -g @angular/cli@9.1.7 but  I still get the same error. However , later I tryed removing  
npm install -g @angular/cli@9.1.7
npm install @angular-devkit/build-angular
those two lines after npm ci and that worked. Before, those two install it wouldnt work. Anyways, it seems solved for now. Thanks for the help. 
Nicolas
